ktor-utils/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 }
 
 kotlin {
-    createCInterop("threadUtils", nixTargets()) {
+    createCInterop("threadUtils", nixTargets() - androidNativeTargets()) {
```

Comment:
The problem exists only for this module where I exclude android native targets from C interop. Maybe it is related

Reply:
we could try to split this interop per related targets.
E.g create `threadUtilsLinux` and `threadUtilsDarwin` with distinct sets of targets (or even different `def` files) - may be there is some bug in commonizer logic

Reply:
Hey @osipxd!
I was able to overcome the issue here: https://github.com/ktorio/ktor/pull/4236/commits/bbd3151262f4c533d53995c78f19e89bc3fcff7f
Previously failed build is now [green](https://ktor.teamcity.com/buildConfiguration/Ktor_AllocationTests/282160) :)